aaelf32/aaelf32.rst

```diff
@@ -0,0 +1,3317 @@
+-----------------------
+
+The ABI for the Arm Architecture uses the following terms and abbreviations.
+
+AAPCS
+   Procedure Call Standard for the Arm Architecture
+
+ABI
+   Application Binary Interface:
+
+   1. The specifications to which an executable must conform in order to
+      execute in a specific execution environment. For example, the
+      Linux ABI for the Arm Architecture.
+
+   2. A particular aspect of the specifications to which independently
+      produced relocatable files must conform in order to be
+      statically linkable and executable.  For example, the C++ ABI
+      for the Arm Architecture, the Run-time ABI for the Arm
+      Architecture, the C Library ABI for the Arm Architecture.
+
+AEABI
+   (Embedded) ABI for the Arm architecture (this ABI…)
+
+Arm-based
+   ... based on the Arm architecture ...
+
+core registers
+   The general purpose registers visible in the Arm architecture’s
+   programmer’s model, typically r0-r12, SP, LR, PC, and CPSR.
+
+EABI
+   An ABI suited to the needs of embedded, and deeply embedded (sometimes
+   called free standing), applications.
+
+Q-o-I
+   Quality of Implementation – a quality, behavior, functionality, or
+   mechanism not required by this standard, but which might be provided by
+   systems conforming to it. Q-o-I is often used to describe the
+   tool-chain-specific means by which a standard requirement is met.
+
+VFP
+   The Arm architecture’s Floating Point architecture and instruction set.
+   In this ABI, this abbreviation includes all floating point variants
+   regardless of whether or not vector (V) mode is supported.
+
+Acknowledgements
+----------------
+
+This specification has been developed with the active support of the following
+organizations. In alphabetical order: Arm, CodeSourcery, Intel, Metrowerks,
+Montavista, Nexus Electronics, PalmSource, Symbian, Texas Instruments, and Wind
+River.
+
+.. raw:: pdf
+
+   PageBreak
+
+Scope
+=====
+
+This specification provides the processor-specific definitions required by ELF
+[SCO-ELF_] for Arm based systems.
+
+The ELF specification is part of the larger System V ABI specification where it
+forms chapters 4 and 5.  However, the specification can be used in isolation as
+a generic object and executable format.
+
+`Platform Standards`_ of this document covers ELF related matters that are
+platform specific.  Most of this material is related to the Base Platform ABI.
+
+`Object Files`_ and `Program Loading and Dynamic Linking`_ of this document are
+structured to correspond to chapters 4 and 5 of the ELF specification.
+Specifically:
+
+* `Object Files`_ covers object files and relocations
+
+* `Program Loading and Dynamic Linking`_ covers program loading and dynamic linking.
+
+There are several drafts of the ELF specification on the SCO web site.  This
+specification is based on the December 2003 draft, which was the most recent
+stable draft at the time this specification was developed.
+
+Platform Standards
```

Comment:
In the rest of the document, could you add page breaks between the different chapters? :
```
.. raw:: pdf

   PageBreak
```

Reply:
Done.